bugsnag-plugin-android-anr/src/main/java/com/bugsnag/android/AnrPlugin.kt

```diff
@@ -14,13 +15,15 @@ internal class AnrPlugin : BugsnagPlugin {
     private lateinit var client: Client
     private val collector = AnrDetailsCollector()
 
-    private external fun enableAnrReporting()
+    private external fun enableAnrReporting(callPreviousSigquitHandler: Boolean)
     private external fun disableAnrReporting()
 
     override fun loadPlugin(client: Client) {
-        enableAnrReporting()
-        Logger.info("Initialised ANR Plugin")
-        this.client = client
+        Handler(Looper.getMainLooper()).post(Runnable {
```

Comment:
This could use a comment to indicate why it must be called from the main thread.